<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MenuWindow</class>
 <widget class="QWidget" name="MenuWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="Button" name="cExit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>290</y>
     <width>256</width>
     <height>26</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>256</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>256</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="Button" name="cOptions">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>248</y>
     <width>256</width>
     <height>26</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>256</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>256</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Options</string>
   </property>
  </widget>
  <widget class="Button" name="cStartCareer">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>122</y>
     <width>256</width>
     <height>26</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>256</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>256</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Start New Career</string>
   </property>
  </widget>
  <widget class="Button" name="cCityConstruction">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>206</y>
     <width>256</width>
     <height>26</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>256</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>256</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>City Construction Kit</string>
   </property>
  </widget>
  <widget class="Button" name="cLoadGame">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>164</y>
     <width>256</width>
     <height>26</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>256</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>256</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Load Saved Game</string>
   </property>
  </widget>
  <widget class="QLabel" name="cBackground">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>480</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>cBackground</zorder>
  <zorder>cExit</zorder>
  <zorder>cOptions</zorder>
  <zorder>cStartCareer</zorder>
  <zorder>cCityConstruction</zorder>
  <zorder>cLoadGame</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Button</class>
   <extends>QPushButton</extends>
   <header>widget/button.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
